core/services/relay/evm/tron_transmitter.go

```diff
@@ -0,0 +1,92 @@
+package evm
```

Comment:
I'd add an explicit TODO to move to chainlink-tron once chainlink-evm is extracted